src/Unosquare.RaspberryIO.Peripherals/Accelerometer GY521/EnumGY521.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace Unosquare.RaspberryIO.Peripherals
+{
+    /// <summary>
+    /// Enum FSSEL.
+    /// </summary>
+    public enum FSSEL
+    {
+        /// <summary>
+        /// Gyroscope Full Scale Range 250
+        /// </summary>
+        FSR250 = 0x00,
```

Comment:
"By default, the first enumerator has the value 0, and the value of each successive enumerator is increased by 1." You don't need to explicitly declare enum values in this case.